graph_api/node/node_router.py

```diff
@@ -60,6 +63,19 @@ async def get_nodes(self, label: str, response: Response):
 
         return nodes
 
+    @router.post("/nodes_query", tags=["nodes"], response_model=NodeRowsOut)
```

Comment:
Why this is `post` not `get`?

Reply:
I discussed it with @terzawad and POST request with JSON body have benefits:
- clear, human readable JSON format (no GET array parameters)
- query is not limited to 2048 characters

I know that in pure REST API it should be GET.

Reply:
Got it 👍